test/rp66.cpp

```diff
@@ -460,3 +460,25 @@ TEST_CASE(
 
     lfp_close(cfile);
 }
+
+TEST_CASE(
+    "Layered rp66 closes correctly",
+    "[visible envelope][rp66]") {
+    const auto file = std::vector< unsigned char > {
+        /* First Visible Envelope - outer layer*/
+        0x00, 0x0A,
+        0xFF, 0x01,
+        /* First Visible Envelope - inner layer*/
+        0x00, 0x06,
+        0xFF, 0x01,
+        /* Body */
+        0x01, 0x02,
+    };
+
+    auto* mem = lfp_memfile_openwith(file.data(), file.size());
+    auto* outer = lfp_rp66_open(mem);
+    auto* inner = lfp_rp66_open(outer);
```

Comment:
I think we have different ideas about what inner and outer mean :smile: 
At least in the contents themselves you use my understanding where outer is outer and inner is inner